### PR TITLE
videojs-standard Tech

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Makes `src/js/tech/` pass videojs-standard.

Previous discussion in #3462  

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tech/html5.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ import * as Url from '../utils/url.js';
 import * as Fn from '../utils/fn.js';
 import log from '../utils/log.js';
 import tsml from 'tsml';
-import TextTrack from '../../../src/js/tracks/text-track.js';
```
Comment (gkatsev) [inline src/js/tech/html5.js]: Is this being included elsewhere?

File: src/js/tech/html5.js [hunk 2]
```diff
@@ -919,12 +1000,12 @@ class Html5 extends Tech {
...
   removeRemoteTextTrack(track) {
-    if (!this['featuresNativeTextTracks']) {
+    if (!this.featuresNativeTextTracks) {
       return super.removeRemoteTextTrack(track);
     }
 
-    let tracks, i;
-
+    let tracks;
+    let i;
```
Comment (gkatsev) [inline src/js/tech/html5.js]: should we just move this variable to line 1017?

File: src/js/tech/html5.js [hunk 3]
```diff
@@ -1011,8 +1092,8 @@ Html5.nativeSourceHandler.canPlayType = function(type){
  * @param  {Object} options The options passed to the tech
  * @return {String}         'probably', 'maybe', or '' (empty string)
  */
-Html5.nativeSourceHandler.canHandleSource = function(source, options){
-  var match, ext;
+Html5.nativeSourceHandler.canHandleSource = function(source, options) {
+  let ext;
```
Comment (gkatsev) [inline src/js/tech/html5.js]: this could be moved to line 1103?

File: src/js/tech/tech.js [hunk 4]
```diff
@@ -10,10 +10,8 @@ import HTMLTrackElementList from '../tracks/html-track-element-list';
 import mergeOptions from '../utils/merge-options.js';
 import TextTrack from '../tracks/text-track';
 import TextTrackList from '../tracks/text-track-list';
-import VideoTrack from '../tracks/video-track';
```
Comment (gkatsev) [inline src/js/tech/tech.js]: are these included elsewhere?

File: src/js/tech/html5.js (hunk 1 above)
Comment (misteroneill) [inline src/js/tech/html5.js]: Doesn't look like it. I'll bring it back.

Comment (misteroneill) [inline src/js/tech/html5.js]: Scratch that: `src/js/video.js` imports it.

File: src/js/tech/html5.js (hunk 2 above)
Comment (misteroneill) [inline src/js/tech/html5.js]: 👍

File: src/js/tech/html5.js (hunk 3 above)
Comment (misteroneill) [inline src/js/tech/html5.js]: 👍

File: src/js/tech/tech.js (hunk 4 above)
Comment (misteroneill) [inline src/js/tech/tech.js]: Yeah, looks like `src/js/video.js` imports it.

**Comment by BrandonOCasey** [comment]
LGTM

File: src/js/tech/tech.js (hunk 4 above)
Comment (gkatsev) [inline src/js/tech/tech.js]: What about making that block of un-named imports in `player.js` the canonical place for these types of imports?

Comment (misteroneill) [inline src/js/tech/tech.js]: 👍

**Pull request merged**